Adafruit_DAP.h

```diff
@@ -205,13 +205,15 @@ class Adafruit_DAP {
 // DAP for SAM
 class Adafruit_DAP_SAM : public Adafruit_DAP {
 public:
-  Adafruit_DAP_SAM(void){};
+  Adafruit_DAP_SAM(void) : locked(false) {};
   ~Adafruit_DAP_SAM(void){};
 
   static device_t devices[];
   device_t target_device;
 
   bool select(uint32_t *id);
+  void finishReset(void);
+  bool locked;
```

Comment:
variable should be move up to line 213 before member function

Reply:
Will do!